### ISSUE TITLE
Chapter 6 Typo

### ISSUE THREAD

**Issue by rickiepark** [rasbt/python-machine-learning-book-2nd-edition#80]
(P186) In note, "to load the **Wine** dataset from ..." should be "to load the **breast cancer** dataset from ..."

**Comment by rickiepark** [comment]
(p194) In first sentence, "we initialized the StratifiedKfold iterator..." should be "we initialized the StratifiedKFold iterator...".

**Comment by rasbt** [comment]
Good catch in both cases. Thanks a lot! Will be fixed in the upcoming third edition.
